frontend/src/features/coverLetter/components/CoverLetterContent.tsx

```
@@ -0,0 +1,156 @@
+    <div
+      ref={containerRef}
+      onMouseUp={handleMouseUp}
+      className='relative ml-12 min-h-0 w-full flex-1 px-4'
+      style={{
+        whiteSpace: 'pre-wrap',
+        overflowY: selection ? 'hidden' : 'auto',
+      }}
+    >
+      <div
+        ref={contentRef}
+        contentEditable={true}
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)